man/loader.conf.xml

```diff
@@ -215,6 +215,18 @@
         Currently, only x86 is supported, where it uses the PC speaker.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>auto-enroll-keys</term>
+
+        <listitem><para>Takes a boolean argument. Enable or disable (the default) the auto enrollement
+        of secure boot keys. If this is enabled, the UEFI firmware is in setup-mode and a directory named
+        'auto' exists on the ESP under <filename>/loader/keys</filename> then the db, KEK and PK secure boot
+        variables are automatically loaded.
+
+        Note that even if it disabled, the bootloader will show entries allowing the user to manually load
```

Comment:
this looks like you wanted another paragraph here. If so you need to end the previous one with `</para>` and open a new one here with `<para>`...